DashAI/back/explainability/local_explainer.py

```diff
@@ -0,0 +1,12 @@
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+
+
+class LocalExplainer:
+    def __init__(self) -> None:
+        pass
+
+    def fit(self, x: DashAIDataset):
```

Comment:
Wouldn't it be better to call it x_train?
```suggestion
    def fit(self, x_train: DashAIDataset):
```